src/features/ITGovernance/Requester/SystemIntake/ContactDetails/index.tsx

```diff
@@ -134,11 +133,12 @@ const ContactDetails = ({ systemIntake }: ContactDetailsProps) => {
     shouldValidate: boolean = false
   ) => {
     if (!isDirty) return callback();
+    const values = watch();
```

Comment:
Is there a reason you moved this into the submit function? Also it should probably still get the actual field instead of retrieving all values (`watch('isPresent')`).

Going to approve as-is because it works, but just wanted to point out best practice for RHF.